diff --git a/prettyprinter/src/Prettyprinter/Internal.hs b/prettyprinter/src/Prettyprinter/Internal.hs
index 6297c4f..1ec8e34 100755
--- a/prettyprinter/src/Prettyprinter/Internal.hs
+++ b/prettyprinter/src/Prettyprinter/Internal.hs
@@ -1581,7 +1581,7 @@ alterAnnotationsS re = go []
         SText l t rest    -> SText l t (go stack rest)
         SLine l rest      -> SLine l (go stack rest)
         SAnnPush ann rest -> case re ann of
-            Nothing   -> go (Remove:stack) rest
+            Nothing   -> go stack rest
             Just ann' -> SAnnPush ann' (go (DontRemove:stack) rest)
         SAnnPop rest      -> case stack of
             []                -> panicPeekedEmpty
