diff --git a/prettyprinter/src/Prettyprinter/Internal.hs b/prettyprinter/src/Prettyprinter/Internal.hs
index 6297c4f..75fc82f 100755
--- a/prettyprinter/src/Prettyprinter/Internal.hs
+++ b/prettyprinter/src/Prettyprinter/Internal.hs
@@ -1663,7 +1663,7 @@ fuse depth = go
         Nest 0 x         -> go x
         Nest i x         -> Nest i (go x)
 
-        Annotated ann x -> Annotated ann (go x)
+        Annotated _ Empty -> Empty
 
         FlatAlt x1 x2 -> FlatAlt (go x1) (go x2)
         Union x1 x2   -> Union (go x1) (go x2)
