diff --git a/prettyprinter/src/Prettyprinter/Internal.hs b/prettyprinter/src/Prettyprinter/Internal.hs
index 6297c4f..6c1c4cc 100755
--- a/prettyprinter/src/Prettyprinter/Internal.hs
+++ b/prettyprinter/src/Prettyprinter/Internal.hs
@@ -1894,7 +1894,7 @@ remainingWidth lineLength ribbonFraction lineIndent currentColumn =
     columnsLeftInLine = lineLength - currentColumn
     columnsLeftInRibbon = lineIndent + ribbonWidth - currentColumn
     ribbonWidth =
-        (max 0 . min lineLength . floor)
+        (max 0 . min lineLength . round)
             (fromIntegral lineLength * ribbonFraction)
 
 -- $ Test to avoid surprising behaviour
